docs/splatalogue/splatalogue.rst

```diff
@@ -35,51 +29,53 @@ To search within this list for a particular species, you can use regular express
 
 .. code-block:: python
 
-   >>> CO_containing_species = Splatalogue.get_species_ids('CO')
+   >>> CO_containing_species = Splatalogue.get_species_ids(species_regex='CO')
    >>> len(CO_containing_species)
-   91
-   >>> just_CO = Splatalogue.get_species_ids(' CO ') # note the spaces
+   105
+   >>> just_CO = Splatalogue.get_species_ids(species_regex=' CO ') # note the spaces
    >>> len(just_CO)
    4
-   >>> just_CO  # includes different vibrationally excited states
-   {u'02812 CO v = 0 - Carbon Monoxide': u'204',
-    u'02813 CO v = 1 - Carbon Monoxide': u'990',
-    u'02814 CO v = 2 - Carbon Monoxide': u'991',
-    u'02815 CO v = 3 - Carbon Monoxide': u'1343'}
+   >>> assert just_CO == {'02812 CO v = 0 - Carbon Monoxide': '204',
```

Comment:
why the asserts in the documentation?

Reply:
I should've left myself a note about this... iirc, there was a problem with the non-assert version. I'll remove and see what happens.

Reply:
Looks like tests pass. I bet it was some local configuration error.